src/BaseTemplates/WebAPI/web.config

```diff
@@ -9,6 +9,6 @@
     <handlers>
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified"/>
     </handlers>
-    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" startupTimeLimit="3600" stdoutLogEnabled="false"/>
+    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="false"/>
```

Comment:
Not sure if this is a problem but empty directories is not picked up by git. So even if the user generates a template with the empty wwwroot folder, if they clone it somewhere else, they will be missing that directory.